### PR TITLE
Document the available routes

### LINKED ISSUE #380: Document information architecture (available routes)
### Feature description

After discussing our strategy to define the available routes in Saiba and coming to an agreement, we need to document it for future reference.

### Dependencies

Depends on:

- https://github.com/Qiskit/saiba/issues/316

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by abdonrd**
## Changes

Document the available routes

Fix #380

## Implementation details

I just created a README.md inside of the `pages/` folder.

### CODE REVIEW

File: pages/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+### Authentication
+
+- `/signin`: Public sign in
+- `/signin/private-experience`: Private sign in
+
+---
+
+- The `courseSlug` is unique like: `quantum-machine-learning`
+
+  - We can auto-generate it based on the title
```
Comment (Tansito) [inline pages/README.md:38]: One question here @abdonrd , is this an idea that we had or are we thinking to do it at some point?

**Review by Tansito — APPROVED** [review]
LGTM! I don't know if @eddybrando has other opinion but I really liked the place that you find to put this information 💯

**Comment by eddybrando** [comment]
> LGTM! I don't know if @eddybrando has other opinion but I really liked the place that you find to put this information 💯

I think it's a very natural place to have it documented. Right next to the relevant files. 💪🧠

File: pages/README.md (hunk 1 above)
Comment (abdonrd) [reply pages/README.md:38]: Oh, good point! I will add it here: https://github.com/Qiskit/saiba/issues/381

Comment (Tansito) [reply pages/README.md:38]: I would keep editable anyway in case we want/need to modify something.

**Pull request merged**